hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestConcatenatedCompressedInput.java

```diff
@@ -705,7 +690,7 @@ private static String unquote(String in) {
    * @param args
    * @throws Exception
    */
-  public static void main(String[] args) throws Exception {
+  /*public static void main(String[] args) throws Exception {
```

Comment:
Should we keep this main entry point? I've never run it myself, but I wonder if someone else found it useful.

Reply:
Thank you for noticing this issue! I will fix it.